src/RatingAPILayer.jsx

```diff
@@ -70,7 +70,8 @@ class RatingAPILayer extends React.PureComponent {
       fractions,
       direction,
       start,
-      stop
+      stop,
+      ...other
```

Comment:
Please remove passing other props. Let's open an issue to discuss any specific additions you'd like to see to the API. I don't really like the idea of mass passing of undocumented props.